themes/prism.css

```diff
@@ -4,8 +4,8 @@
  * @author Lea Verou
  */
 
-code[class*="language-"],
-pre[class*="language-"] {
+code[class*='language-'],
```

Comment:
Let's add an override for CSS files? It seems double quotes are more common (which makes sense, as they mirror HTML, where double quotes are far more common).

Reply:
Done!